File: Benchmarking/stats/tester/euler1/euler1.java
public class euler1 {
	public static void main (String[] args)
	{
		//f1(Integer.parseInt(args[0]));
		f1(Long.parseLong(args[0]));
	}

	private static long f1 (long n)
	{
		long sum = 0;
		for (long i = 1; i < n; i++)
		{
			if (i % 3 == 0 || i % 5 == 0)
			{
				sum += i;
			}
		}
		return sum;
	}
}

File: Benchmarking/stats/tester/euler1/euler1.cpp
//
//  main.cpp
//  Euler1
//
//  Created by Sonni on 19/05/16.
//  Copyright © 2016 Sonni Jensen. All rights reserved.
//

#include <iostream>

void f1(unsigned long long n)
{
    unsigned long long sum = 0;
    for (unsigned long long i = 0; i < n; i++)
    {
        if (i % 3 == 0 || i % 5 == 0)
            sum += i;
    }

    //std::cout << sum << std::endl;
}

int main(int argc, const char * argv[]) {
    f1(atoi(argv[1]));
    return 0;
}
